docs/reference/getting-started.asciidoc

```diff
@@ -545,7 +545,7 @@ For the curious, I generated this data from http://www.json-generator.com/[`www.
 [float]
 === Loading the Sample Dataset
 
-You can download the sample dataset (accounts.json) from https://github.com/bly2k/files/blob/master/accounts.zip?raw=true[here]. Extract it to our current directory and let's load it into our cluster as follows:
+You can download the sample dataset (accounts.json) from https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/src/test/resources/accounts.json[here]. Extract it to our current directory and let's load it into our cluster as follows:
```

Comment:
I make it consistent with other versions, can we make the link
https://github.com/elastic/elasticsearch/blob/master/docs/src/test/resources/accounts.json?raw=true

Reply:
@mayya-sharipova Thanks for the link. I will update this.